modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -183,15 +331,142 @@ function acquia_cms_site_studio_update_8003() {
 }
 
 /**
- * Implements hook_uninstall().
- *
- * Reset page field body's label and descritption to default.
+ * Update role permissions.
  */
-function acquia_cms_site_studio_uninstall() {
-  $config = \Drupal::service('config.factory')->getEditable('field.field.node.page.body');
-  if (!empty($config)) {
-    $config->set('label', 'Body');
-    $config->set('description', '');
-    $config->save();
+function acquia_cms_site_studio_update_8004() {
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('acquia_cms_site_studio')) {
```

Comment:
@rajeshreeputra Same as mentioned below i.e de-dupe code and condition check not required.
Also, can we make the code more readable ? Ex: 

```
// Define basic permission.
$site_studio_basic_permissions = [
  'access markup',
  'access seo',
  'access styles',
  'access tokens',
  //....and so on.
];

$basic_component_perimissions = [
  'access accordion_sections cohesion_helper_category group',
  'access card_sections cohesion_helper_category group',
  'access cpt_cat_accordion_components cohesion_component_category group',
  'access cpt_cat_basic_components cohesion_component_category group',
];

$role_permissions = [
  'content_author' => $site_studio_basic_permissions,
  'developer' => array_merge($site_studio_basic_permissions, $basic_component_perimissions),
  //.. and so on for other roles.
];
```

Reply:
added comment above permissions